.github/workflows/cherry-pick-from-release-to-develop.yml

```diff
@@ -47,113 +41,9 @@ jobs:
-                  CHERRY_PICK_COMMIT=$(git rev-parse HEAD)
-                  echo "cherryPickCommit=$CHERRY_PICK_COMMIT" >> $GITHUB_ENV
-
-            - name: Get Original Author
-              id: get-author
-              if: env.shouldCherryPick == 'true'
-              run: |
-                  ORIGINAL_AUTHOR=$(git log -1 --pretty=format:'%an <%ae>' ${{ github.event.pull_request.merge_commit_sha }})
-                  echo "Original author: $ORIGINAL_AUTHOR"
-                  echo "originalAuthor=$ORIGINAL_AUTHOR" >> $GITHUB_ENV
-
-            - name: Cherry-pick commits
-              id: commit-cherry-pick
-              if: env.shouldCherryPick == 'true'
-              run: |
-                  # Checkout the desired branch and cherry-pick the commit
-                  git checkout ${{ env.TARGET_BRANCH }}
-                  git checkout -b ${{ env.newBranchName }}
-                  OUTPUT=$(git cherry-pick ${{ env.cherryPickCommit }} || true)
-                  
-                  # Handle conflicts
-                  CONFLICTED_FILES=$(git diff --name-only --diff-filter=U | awk 'ORS="\\\\n"' | sed 's/\\\\n$/\\n/')
-                  echo "Captured conflicted files: $CONFLICTED_FILES"
-                  if [[ "$OUTPUT" == *"CONFLICT"* ]]; then
-                      # Commit the remaining conflicts
-                      git add .
-                      git commit --author "${{ env.originalAuthor }}" -am "Commit with unresolved merge conflicts outside of ${{ env.SUBMODULE_NAME }}"
-                  else
-                      git commit --author "${{ env.originalAuthor }}" --amend --no-edit
-                  fi
-                  
-                  # Push branch and remove temp
-                  git push origin ${{ env.newBranchName }} || (echo "Failed to push to origin" && exit 1)
-                  echo "conflictedFiles=$CONFLICTED_FILES" >> $GITHUB_ENV
-                  if [[ -n "${{ env.SUBMODULE_NAME }}" ]]; then
-                  git branch -D temp-branch-for-cherry-pick
-                  fi
-
-            - name: Create PR
-              if: env.shouldCherryPick == 'true'
-              env:
-                  PR_TITLE: ${{ github.event.pull_request.title }}
-                  PR_BRANCH: ${{ env.newBranchName }}
-                  PR_ASSIGNEE: ${{ github.event.pull_request.user.login }}
-              run: |
-                  PR_BODY=$(echo -e "Cherry pick from the original PR: \n- #${{ github.event.pull_request.number }}\n\n---- \n\n ⚠️ Conflicts during cherry-pick:\n${{ env.conflictedFiles }}\n\n${{ github.event.pull_request.body }}")
-                  gh pr create --title "$PR_TITLE" --body "$PR_BODY" --base ${{ env.TARGET_BRANCH }} --head "$PR_BRANCH" --label "cherry-pick" --assignee "$PR_ASSIGNEE"
+            - name: Cherry pick to develop
+              uses: wireapp/action-auto-cherry-pick@v1.0.0
+              with:
+                  target-branch: develop
+                  pr-labels: 'cherry-pick'
```

Comment:
I think we don't want the default suffix in title, so need to add `pr-title-suffix: ''"`